src/components/ArtifactDataTable.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <p-layout-table class="artifact-data-table">
+    <template #header-end>
+      <span class="artifact-data-table__results-count">
+        <template v-if="filteredCount !== dataCount">
+          <span class="artifact-data-table__count">{{ filteredCount }}</span> of
+        </template>
+        <span class="artifact-data-table__count">{{ dataCount }}</span>
+        {{ toPluralString(localization.info.item, dataCount) }}
+      </span>
```

Comment:
I feel like this is different than our other tables. Don't we usually just say x items and lower the number as users search?

Reply:
Yeah I was trying to make a pretty distinct break from other tables since this is for user-supplied content; the expectations are necessarily different for how this will operate

Reply:
This feels like a pattern break that is going to make our other tables feel off compared to this one. My vote would be to decide if this is the pattern we want to follow for counting results and update the other places as well. I think this is a good pattern. But I wanna keep the user experience consistent.